hw/dv/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Design Verification"
```

Comment:
Perhaps `Common Design Verification Collateral`?

Reply:
Thanks, I've changed this.